README.md

```diff
@@ -2399,6 +2399,7 @@ Signed,
 - Yazeed AlKhalaf ([@YazeedAlKhalaf](https://github.com/YazeedAlKhalaf)), Flutter Developer and Youngest Participant in Hajj Hackathon, Saudi Arabia
 - Jan Klinge ([@Klingens13](https://github.com/Klingens13)), Student, doing almost everything from hardware hacking to software developing, Germany
 - Joseph Robinson ([@JosephWorks](https://github.com/josephworks)), Software Developer, Engineer, **Master of Technology**, GitHub Education Student Member
-- Anubhav Madhav ([@AnubhavMadhav] (https://github.com/AnubhavMadhav)), Student at Indian Institute of Information Technology, Vadodara, India.
+- Anubhav Madhav ([@AnubhavMadhav](https://github.com/AnubhavMadhav)), Student at Indian Institute of Information Technology, Vadodara, India.
```

Comment:
Thanks for fixing this, but it should be a different PR. A script handles the process so you should only modify the end of the file.